README.md

```diff
@@ -42,6 +42,55 @@ task assets build
 If Bash is available (Linux and Windows while on Git Bash), the commands will
 run in Bash, otherwise will fallback to `cmd` (on Windows).
 
+### Variables
+
+```yml
+build:
+  deps: [setvar]
+  cmds:
+  - echo "{{prefix}} {{THEVAR}}"
+  variables:
+    prefix: "Path:"
+
+setvar:
+  cmds:
+  - echo "{{PATH}}"
+  set: THEVAR
```

Comment:
`set` is the property with the variable name. I searched for a valid name of the most simplistic task to set a variable.